### DISCUSSION TITLE
[Request] RISC-V architecture

### DISCUSSION THREAD

**Discussion by jackghg** [NVIDIA/open-gpu-kernel-modules#242] [Ideas]
Support for risc-v arch please.

**Comment by RealAstolfo** [comment]
please change label from bug to enhancement. this is not a bug. it has been explicitly said in the README that the supported arches are `TARGET_ARCH=aarch64|x86_64`

**Comment by Leseratte10** [comment]
The label is auto-generated by Github for issue reports, there is no way for the reporter to pick a label other than "bug". This needs to be fixed by the Nvidia team to allow for other report types; in the meantime someone with commit access has to correct all these labels for new reports.

**Comment by PAR2020** [comment]
We just added a Feature Request template, which should help. Our team is going through and manually changing any other issues affected. Thank you for your patience.

**Comment by mtijanic** [comment]
#152  :)
